test/e2e/presto_prometheus_connector_up_test.go

```diff
@@ -2,18 +2,17 @@ package e2e
 
 import (
 	"bytes"
-	"github.com/kube-reporting/metering-operator/test/reportingframework"
-	"github.com/stretchr/testify/require"
 	"os"
 	"os/exec"
 	"strings"
 
+	"github.com/kube-reporting/metering-operator/test/reportingframework"
+	"github.com/stretchr/testify/require"
+
 	"testing"
 )
 
 func testPrometheusConnectorWorks(t *testing.T, rf *reportingframework.ReportingFramework) {
-	t.Helper()
```

Comment:
Was this just leftover junk that got missed in a previous revision?

Reply:
Yeah `t.Helper()` will silence any of these test helper logs, and we moved all of the post-install logging into its own file, so I think removing this and capturing the logs from this function is best.